app/controllers/turbo/frames/frame_request.rb

```diff
@@ -1,20 +1,24 @@
 # Turbo frame requests are requests made from within a turbo frame with the intention of replacing the content of just
 # that frame, not the whole page. They are automatically tagged as such by the Turbo Frame JavaScript, which adds a
-# <tt>Turbo-Frame</tt> header to the request. When that header is detected by the controller, we ensure that any
-# template layout is skipped (since we're only working on an in-page frame, thus can skip the weight of the layout), and
-# that the etag for the page is changed (such that a cache for a layout-less request isn't served on a normal request
-# and vice versa).
+# <tt>Turbo-Frame</tt> header to the request.
 #
-# This is merely a rendering optimization. Everything would still work just fine if we rendered everything including the layout.
-# Turbo Frames knows how to fish out the relevant frame regardless.
+# When the <tt>Turbo-Frame</tt> header is present, the request's <a
+# href="https://guides.rubyonrails.org/4_1_release_notes.html#action-pack-variants">variant</a>
+# will be marked <tt>turbo_frame</tt>.
+#
+# As an optimization, applications can declare a layout with the
+# <tt>html+turbo_frame.erb</tt> extension that renders
+# without the navigation and layout elements:
+#
+#     <%= app/views/layouts/application.html+turbo_frame.erb %>
```

Comment:
```suggestion
#     <%# app/views/layouts/application.html+turbo_frame.erb %>
```

Reply:
Would it be possible to generate this view as part of the install generator?

Reply:
What exactly would be in this layout? Seems like we would need a way to customize the turbo_frame_id here if it wraps the rendered content with a turbo_frame, but I am not sure that is the intent.